crates/bevy_gizmos/src/arcs.rs

```diff
@@ -109,3 +109,301 @@ fn arc_inner(
+    /// - rotation
+    /// - color
+    /// - segments
+    ///
+    /// # Arguments
+    ///
+    /// - `center` - The center point around which the arc is drawn.
+    /// - `from` - The starting point of the arc.
+    /// - `to` - The ending point of the arc.
+    ///
+    /// # Examples
+    /// ```
+    /// # use bevy_gizmos::prelude::*;
+    /// # use bevy_render::prelude::*;
+    /// # use bevy_math::prelude::*;
+    /// # use std::f32::consts::PI;
+    /// fn system(mut gizmos: Gizmos) {
+    ///     gizmos.short_arc_3d_between(Vec3::ZERO, Vec3::X, Vec3::Y);
+    ///
+    ///     // This example shows how to modify the default settings
+    ///
+    ///     gizmos.short_arc_3d_between(Vec3::ONE, Vec3::ONE + Vec3::NEG_ONE, Vec3::ZERO)
+    ///        .radius(0.25)
+    ///        .center(Vec3::ZERO)
+    ///        .segments(100)
+    ///        .color(Color::ORANGE);
+    /// }
+    /// # bevy_ecs::system::assert_is_system(system);
+    /// ```
+    ///
+    /// # Notes
+    /// - This method assumes that the points `from` and `to` are distinct from the `center`. If
+    /// the points are coincident with the `center`, the behavior is undefined.
+    /// - The arc is drawn as a portion of a circle with a radius equal to the distance from the
+    /// `center` to `from`. If the distance from `center` to `to` is not equal to the radius, then
+    /// the results will behave as if this were the case
+    #[inline]
+    pub fn short_arc_3d_between(
+        &mut self,
+        center: Vec3,
+        from: Vec3,
+        to: Vec3,
+    ) -> Arc3dBuilder<'_, 'w, 's, T> {
+        self.arc_from_to(center, from, to, |x| x)
+    }
+
+    /// Draws the longest arc between two points (`from` and `to`) relative to a specified `center` point.
+    ///
+    /// The arc can be modified via the arc builder pattern. The following properties can be adjusted:
+    ///
+    /// - radius
+    /// - center
+    /// - rotation
+    /// - color
+    /// - segments
+    ///
+    /// # Arguments
+    /// - `center` - The center point around which the arc is drawn.
+    /// - `from` - The starting point of the arc.
+    /// - `to` - The ending point of the arc.
+    ///
+    /// # Examples
+    /// ```
+    /// # use bevy_gizmos::prelude::*;
+    /// # use bevy_render::prelude::*;
+    /// # use bevy_math::prelude::*;
+    /// # use std::f32::consts::PI;
+    /// fn system(mut gizmos: Gizmos) {
+    ///     gizmos.long_arc_3d_between(Vec3::ZERO, Vec3::X, Vec3::Y);
+    ///
+    ///     // This example shows how to modify the default settings
+    ///     gizmos.long_arc_3d_between(Vec3::ONE, Vec3::ONE + Vec3::NEG_ONE, Vec3::ZERO)
+    ///        .radius(0.25)
+    ///        .center(Vec3::ZERO)
+    ///        .segments(100)
+    ///        .color(Color::ORANGE);
+    /// }
+    /// # bevy_ecs::system::assert_is_system(system);
+    /// ```
+    ///
+    /// # Notes
+    /// - This method assumes that the points `from` and `to` are distinct from the `center`. If
```

Comment:
I'm a bit nervous about using "behavior is undefined" here (and above): it's not UB in the compiler sense and we may confuse readers. Maybe "graphical glitches may occur"?

Reply:
Thanks for spotting this!

I just tested these cases (after all we're also part scientists, right? :D). It seems like nothing happens at all. We should probably communicate this to the user.

Do you think an early return might make sense? I'm not really sure. On the one hand we could save some compute in the case that we wouldn't draw anything anyways. On the other hand this is an edge case and adding two checks for every non-edge case might be needless overhead.

I think I'm just going for changing the comment for now, but please let me know what you think.

Reply:
I think let's just change the comment and leave the behavior for now. I'm really not concerned about the performance in pathological cases and if it doesn't crash I'm quite happy with it as is.